augur/tree.py

```diff
@@ -348,8 +356,11 @@ def register_arguments(parser):
                                 help="number of threads to use; specifying the value 'auto' will cause the number of available CPU cores on your system, if determinable, to be used")
     parser.add_argument('--vcf-reference', type=str, help='fasta file of the sequence the VCF was mapped to')
     parser.add_argument('--exclude-sites', type=str, help='file name of one-based sites to exclude for raw tree building (BED format in .bed files, second column in tab-delimited files, or one position per line)')
-    parser.add_argument('--tree-builder-args', type=str, default='', help='extra arguments to be passed directly to the executable of the requested tree method (e.g., --tree-builder-args="-czb")')
-
+    parser.add_argument('--tree-builder-args', type=str, help=f"""arguments to pass to the tree builder overriding the default arguments except for input alignment path, number of threads, and substitution model (e.g., --tree-builder-args="-czb" for IQ-TREE).
```

Comment:
I recommend adding a line about assignment operator usage for `--tree-builder-args`, eg:

    Use assignment operator (e.g., --tree-builder-args="-czb") to avoid unexpected errors.

Reply:
Good call. I added this line to the help.